test/parallel/test-tls-passphrase.js

```diff
@@ -37,15 +37,17 @@ assert(Buffer.isBuffer(cert));
 assert.strictEqual(typeof passKey.toString(), 'string');
 assert.strictEqual(typeof cert.toString(), 'string');
 
+const onSecureConnect = common.mustCall(function() {
+  this.end();
+}, 22);
```

Comment:
While this is okay, its stack trace won't tell you which `tls.connect()` call fails. You could turn this into a function factory, er, function:
```js
function onSecureConnect() {
  return common.mustCall(function() { this.end(); });
}
```
Then call it like this:
```js
tls.connect({ /* ... */ }, onSecureConnect());
```

Reply:
Ok, will apply suggestion in a bit.